### ISSUE TITLE
TW-R8 ETHOS 1.5.18 Firmware TW-R8 1.0.9 OTA Unsupported RX

### ISSUE THREAD

**Issue by plesogor** [FrSkyRC/ETHOS-Feedback-Community#4671]
TW-R8 ETHOS 1.5.18 Firmware TW-R8 1.0.9 OTA Unsupported RX

Tried this today on X20RS, new TW-R8, Unsupported RX error when flashing using OTA. Directly connected to Radio flashed fine.

**Comment by glipski50** [comment]
It’s a known RF bug. There‘s a beta firmware update for ISRM and receiver available that fixes this.
https://github.com/FrSkyRC/RF_Feedback_Group/releases/tag/Beta

But its still beta version.

**Comment by philippe-warnotte** [comment]
It is not normal to have to install an ISRM Beta version Firmware to use the OTA function which must work as it is described in the ETHOS manual from the beginning!
Beta versions are not guaranteed like safe.

**Comment by glipski50** [comment]
I‘m not suggesting to install a beta. I just wanted to say this is a known bug and the  Frsky HF-team is working on it.

**Comment by philippe-warnotte** [comment]
OK but why do we need to install a new ISRM update to make OTA work again, 
when the problem occurred because of an Ethos update? OTA was ok on Ethos 1.5.13

**Comment by Wolepo** [comment]
This has nothing to do with Ethos, but with the new receiver firmware.

**Comment by glipski50** [comment]
See here https://github.com/FrSkyRC/RF_Feedback_Group/issues?q=is%3Aissue+author%3A%40me+is%3Aclosed

**Comment by mawzthefinn** [comment]
> OK but why do we need to install a new ISRM update to make OTA work again, when the problem occurred because of an Ethos update? OTA was ok on Ethos 1.5.13

The ETHOS update added OTA firmware verification (to prevent flashing incorrect firmware to a receiver), the updated firmware builds work with it. 

This isn't a bug so much as out of sync updates.

**Comment by glipski50** [comment]
If you call it bug or out of sync updates does not matter at the end. Using RC is always using a system with transmitter AND receiver, both should work together. And if one of both does not do what it should the system does not work. Look to it from the end user point of view, not from the developers side. From the end user point of view it simply sucks.

**Comment by mawzthefinn** [comment]
> If you call it bug or out of sync updates does not matter at the end. Using RC is always using a system with transmitter AND receiver, both should work together. And if one of both does not do what it should the system does not work. Look to it from the end user point of view, not from the developers side. From the end user point of view it simply sucks.

I agree, it's a sucky situation and the UX is not what we want to see.

I'm just explaining above as to what happened.

**Comment by philippe-warnotte** [comment]
Thank you for your explanation, I understand now. I wonder why this OTA verification is not part of the list of changes in version 1.5.18 BSongis

**Comment by mawzthefinn** [comment]
The change wasn't in 1.5.18, but earlier. 1.5.14 I think (but not listed). There have been complaints for a little while about it.

**Comment by philippe-warnotte** [comment]
Is this firmware FW-RF-Modul-ISRM-Twin-Beta.v3.0.0 just a test version? Is it possible to fly with this version 3 without risk?
Is the file: "TW-ISRM_3.0.0_Beta.frsk" compatible with TX Kavan V20?

**Comment by philippe-warnotte** [comment]
@bsongis-frsky 
Is this firmware FW-RF-Modul-ISRM-Twin-Beta.v3.0.0 just a test version? Is it possible to fly with this version 3 without risk?
Is the file: "TW-ISRM_3.0.0_Beta.frsk" compatible with TX Kavan V20?

**Comment by mawzthefinn** [comment]
I know individuals who are flying 3.0.0 beta now, and I've been beating up on it with no issues. I'd say it's ready to fly on a test aircraft, but I wouldn't risk anything I cared about on it until you have some time in the air with it.
